python/nano/tutorial/notebook/training/tensorflow/accelerate_tensorflow_training_multi_instance.ipynb

```diff
@@ -0,0 +1,255 @@
+    "    }\n",
+    "</style>\n",
+    "\n",
+    "[View the runnable example on GitHub](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/tensorflow/accelerate_tensorflow_training_multi_instance.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accelerate TensorFlow Keras Training using Multiple Instances"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`bigdl.nano.tf.keras.Model` and `bigdl.nano.tf.keras.Sequential` APIs extend `tf.keras.Model` and `tf.keras.Sequential` separately with optimizations. You could replace `tf.keras.Model`/`Sequential` with `bigdl.nano.tf.keras.Model`/`Sequential` and apply the `fit` method with specified `num_processes` to benefit from multi-instance training on a server with multiple CPU cores or sockets, so that the workload can make full use of all CPU cores."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "source": [
+    "To use multiple instances for TensorFlow Keras training, you need to install BigDL-Nano for TensorFlow:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# install the nightly-bulit version of bigdl-nano for tensorflow;\n",
+    "# intel-tensorflow will be installed at the meantime with intel's oneDNN optimizations enabled by default\n",
+    "!pip install --pre --upgrade bigdl-nano[tensorflow]\n",
+    "!source bigdl-nano-init  # set environment variables"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📝 **Note**\n",
+    ">\n",
+    "> Before starting your TensorFlow Keras application, it is highly recommended to run `source bigdl-nano-init` to set several environment variables based on your current hardware. Empirically, these variables will bring big performance increase for most TensorFlow Keras applications on training workloads."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "source": [
+    "> ⚠️ **Warning**\n",
+    "> \n",
+    "> For Jupyter Notebook users, we recommend to run the commands above, especially `source bigdl-nano-init` before jupyter kernel is started, or some of the optimizations may not take effect."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# install dependency for the dataset used in the following example\n",
+    "!pip install tensorflow-datasets"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To benefit from multi-instance training, you need to **import** `Model` **or** `Sequential` **from** `bigdl.nano.tf.keras` **instead of** `tf.keras` **first**. Let's take the `Model` class here as an example:"
```

Comment:
First, import Model or Sequential from bigdl.nano.tf.keras instead of tf.keras first. Let’s take the Model class here as an example:

Reply:
Have revised.